platform/android/MapboxGLAndroidSDK/src/test/java/com/mapbox/mapboxsdk/location/LocationComponentTest.kt

```diff
@@ -97,6 +97,26 @@ class LocationComponentTest {
     Assert.assertEquals(locationEngineRequest, locationComponent.locationEngineRequest)
   }
 
+  @Test
+  fun activateWithDefaultLocationEngineRequestAndOptionsTest() {
+    locationComponent.activateLocationComponent(context, mockk(), true, locationEngineRequest, locationComponentOptions)
+
+    Assert.assertEquals(locationEngineRequest, locationComponent.locationEngineRequest)
+
+    doReturn(mock(TypedArray::class.java)).`when`(context)
```

Comment:
Because you are already providing `locationComponentOptions` you don't need any of below mocks.

Reply:
Fixed